LBHackney-IT/lbh-fss-portal-frontend
# Feature/admin create organisation and service

**Description**

Allow admin user to link/unlink users from organisations 

**Test instructions**

- pull this branch 
- `yarn install`
- navigate to the users tab **/users**
- the first user in the table 'Adah Johnson' is already linked with an organisation, therefore you should see an option to unlink this user from the organisation 

![image](https://user-images.githubusercontent.com/69238887/100095844-08384200-2e53-11eb-90f4-847b73aa7e3f.png)

- the second user in the table 'Adam Goyette' is not yet linked with an organisation, therefore you should see an option to link this user to an organisation using an autocomplete input field, start typing the name of a known organisation e.g. 'Hackett Inc' and suggestions should appear, if you click 'Link organisation' using an organisation which does not exist e.g. 'asdf' you should get an error message   

![image](https://user-images.githubusercontent.com/69238887/100095946-31f16900-2e53-11eb-8acc-5f98fb6add1c.png)

Note - as mock data is returned by the api after link/unlinking an organisation the page will refresh but no updates will be visible 

**Ticket**
https://app.clickup.com/t/a6zwvf
